src/content/articles/why-typescript-doesnt-follow-strict-semantic-versioning.mdx

```diff
@@ -0,0 +1,278 @@
+---
+date: "July 20 2023"
+description: "Why it's not practically feasible -or desirable- for a programming language like TypeScript to use strict semantic versioning for its releases."
+meta: ECMAScript, compiler options target, TSConfig
+---
+
+import { CaptionedImage } from "@site/src/theme/CaptionedImage";
+
+# Why TypeScript Doesn't Follow Strict Semantic Versioning
+
+:::danger
```

Comment:
Do you remove this before publishing after folks have gotten back to you? Seems to undermine the purpose of your blog post :\ 

I think you can opt for a less critical intro/disclaimer. Something like:
I was inspired to write this blog after I read some hot takes on TS semantic versioning. This is not meant to put an end to the discussion but provide some clarity and offer my own perspective. If you have any feedback on how to improve this post, please let me know!

Reply:
I do remove this before publishing, yes 😄 but thanks!